src/main/java/uk/gov/ons/ctp/response/sample/endpoint/SampleEndpoint.java

```diff
@@ -103,14 +106,14 @@ public SampleEndpoint(SampleService sampleService, MapperFacade mapperFacade) {
       return ResponseEntity.badRequest().build();
     }
 
-    SampleSummary sampleSummary;
     try {
-      sampleSummary = sampleService.ingest(file, type);
+      Pair<SampleSummary, Future<Optional<SampleSummary>>> result = sampleService.ingest(file, type);
+
+      return ResponseEntity.created(URI.create("TODO")).body(result.getLeft());
```

Comment:
Shall we see if we can get rid of the TODO? It's very unlikely this will break anything but worth testings